Sources/JSONRequest.swift

```diff
@@ -77,11 +77,14 @@ open class JSONRequest {
     open static var requestTimeout = 5.0
     open static var resourceTimeout = 10.0
     open static var requestCachePolicy: NSURLRequest.CachePolicy = .useProtocolCachePolicy
+    open static var urlSession: URLSession?
 
     open var httpRequest: NSMutableURLRequest? {
         return request
     }
 
+    internal static var requireNetwork = true
```

Comment:
Consider `requestsFailWithoutNetworkAccess = true`, or `networkAccessRequired = true`.. something like that